query_easyjson.go

```diff
@@ -397,6 +397,10 @@ func easyjson90b16446DecodeGithubComObserveincGosnowflake1(in *jlexer.Lexer, out
 			out.Kind = string(in.String())
 		case "operation":
 			out.Operation = string(in.String())
+		case "queryContext":
+			if data := in.Raw(); in.Ok() {
+				in.AddError((out.QueryContext).UnmarshalJSON(data))
+			}
```

Comment:
I guess we're not re-generating these files so all manual modifications in this file are OK?

Reply:
This is not a manual modification. This comes from `make easyjson`

Reply:
_The upstream is not using easyjson. Instead it uses the default go json marshaler / unmarshaler, which is significantly slower_

Reply:
Ack